qa/tasks/cephfs/test_volumes.py

```diff
@@ -3805,6 +3805,50 @@ def test_subvolume_resize_infinite_size_future_writes(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_resize_valid_unit(self):
+        """
+        That a subvolume can be resized when provided with valid value
+        (both int and float accepted) and unit.
+        """
+        readable_values = {"10K": 10240, "200KB": 204800,
```

Comment:
test for only values with unit "B" should also be added.

Reply:
https://github.com/ceph/ceph/compare/705f4f4ef09a2b8e82fb0bcfa4868c6db114ab98..94bdd112f94f6f80f0e86f1aad8807f83576dc41